renku/core/management/dataset/dataset.py

```diff
@@ -0,0 +1,997 @@
+# -*- coding: utf-8 -*-
```

Comment:
Some functions in this file can be private including `set_dataset_images`, most `update_*`, `filter_dataset_files`.